src/onboarding/steps.ts

```diff
@@ -14,6 +14,7 @@ import * as NavigationService from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { RootState } from 'src/redux/reducers'
 import { store } from 'src/redux/store'
+import { getOnboardingExperimentParams } from 'src//onboarding'
```

Comment:
is this `src//` a typo? 🤔

Reply:
:scream: yes! ty

Reply:
surprised that's not caught by the linter

Reply:
me too, very weird